connectors/vault/vaultconn-bl/vault_reader.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2020 IBM Corp.
+// SPDX-License-Identifier: Apache-2.0
+
+package vaultconnbl
+
+import (
+	"fmt"
+	"log"
+	"strings"
+
+	vaultutils "github.com/ibm/the-mesh-for-data/connectors/vault/vault_utils"
+	pb "github.com/ibm/the-mesh-for-data/pkg/connectors/protobuf"
+)
+
+type Server struct {
+}
+
+func (s *Server) GetCredentialsInfo(in *pb.DatasetCredentialsRequest, vault vaultutils.VaultConnection, vaultPathKey string) (*pb.DatasetCredentials, error) {
+	log.Println("Vault connector: GetCredentialsInfo")
```

Comment:
Why are there two log commands here that are very similar? Can the first one be removed?

Reply:
fixed. Thanks.